edi_835_parser/transaction_set/transaction_set.py

```diff
@@ -501,37 +504,47 @@ def serialize_claim(
 			'created_at': None
 		}
 
-		if claim.adjustment:
-
-			remit_adjustments_dict.update({
-					'remit_key': claim.claim.key,
-					'edi_transaction_id_st02': transaction.transaction.transaction_set_control_no,
-					'adjustment_group_code': claim.adjustment.group_code,
-					'adjustment_reason_code': claim.adjustment.reason_code,
-					'adjustment_amount': claim.adjustment.amount,
-					'adjustment_quantity': claim.adjustment.quantity,
-					'adjustment_reason_code2': claim.adjustment.reason_code2,
-					'adjustment_amount2': claim.adjustment.amount2,
-					'adjustment_quantity2': claim.adjustment.quantity2,
-					'adjustment_reason_code3': claim.adjustment.reason_code3,
-					'adjustment_amount3': claim.adjustment.amount3,
-					'adjustment_quantity3': claim.adjustment.quantity3,
-					'adjustment_reason_code4': claim.adjustment.reason_code4,
-					'adjustment_amount4': claim.adjustment.amount4,
-					'adjustment_quantity4': claim.adjustment.quantity4,
-					'adjustment_reason_code5': claim.adjustment.reason_code5,
-					'adjustment_amount5': claim.adjustment.amount5,
-					'adjustment_quantity5': claim.adjustment.quantity5,
-					'adjustment_reason_code6': claim.adjustment.reason_code6,
-					'adjustment_amount6': claim.adjustment.amount6,
-					'adjustment_quantity6': claim.adjustment.quantity6,
-					'created_at': None
-				})
+		if claim.adjustments:
+			
+			for adjustment in claim.adjustments:
+				remit_adjustments_dict.update({
+						'remit_key': claim.claim.key,
+						'edi_transaction_id_st02': transaction.transaction.transaction_set_control_no,
+						'adjustment_group_code': adjustment.group_code,
+						'adjustment_reason_code': adjustment.reason_code,
+						'adjustment_amount': adjustment.amount,
+						'adjustment_quantity': adjustment.quantity,
+						'adjustment_reason_code2': adjustment.reason_code2,
+						'adjustment_amount2': adjustment.amount2,
+						'adjustment_quantity2': adjustment.quantity2,
+						'adjustment_reason_code3': adjustment.reason_code3,
+						'adjustment_amount3': adjustment.amount3,
+						'adjustment_quantity3': adjustment.quantity3,
+						'adjustment_reason_code4': adjustment.reason_code4,
+						'adjustment_amount4': adjustment.amount4,
+						'adjustment_quantity4': adjustment.quantity4,
+						'adjustment_reason_code5': adjustment.reason_code5,
+						'adjustment_amount5': adjustment.amount5,
+						'adjustment_quantity5': adjustment.quantity5,
+						'adjustment_reason_code6': adjustment.reason_code6,
+						'adjustment_amount6': adjustment.amount6,
+						'adjustment_quantity6': adjustment.quantity6,
+						'created_at': None
+					})
 
 		return {'remits_dict': remits_dict, 'remit_payers_dict': remit_payers_dict,
 										'remit_remarks_adjudications_dict': remit_remarks_adjudications_dict,
 										'remit_adjustments_dict': remit_adjustments_dict}
 
+	@staticmethod
+	def find_coverage_amount(claim):
```

Comment:
could you elaborate on what the claim_coverage_amount is?

Reply:
The amount that has the qualifier "AU", I think each claim only has one such value (but correct me if im wrong @lrpauley)  so since now we are storing multiple amounts, we need to loop through and find it,

Reply:
yep it looks like that's the distinction for AU. i think this logic is better suited for a property on the claim loop similar to one of [these methods](https://github.com/CoveHealth/edi-835-parser/blob/main/edi_835_parser/loops/claim.py#L100-L108)